src/middleware/error-handler.ts

```diff
@@ -0,0 +1,33 @@
+import createError, { HttpError } from 'http-errors';
+import { Context, Middleware, Next } from 'koa';
+import { hydra } from 'rdf-namespaces';
+
+const handleHttpError = (error: HttpError, { response }: Context): void => {
+  response.status = error.status;
+  response.body = {
+    '@type': hydra.Status,
+    [hydra.title]: { '@value': response.message, '@language': 'en' },
```

Comment:
at first `response.message` looked magic in comparison to `error.message`, but thanks to the test now clear it's the `Not Found`|`Internal Server Error`|... string.